### DISCUSSION TITLE
Is there a way to use multiple filters and search bar together on same column?

### DISCUSSION THREAD

**Discussion by proasian42** [TanStack/table#2162] [General]
It is my first time using a react-table. I am trying to use multiple filters on some header columns. For example, I want the user to select one or more fruits or to search for fruit on the same column. I would like both filters, multi-select filter and search bar, on the same column.

I couldn't find a way to combine those filters together. I would like to know if there is a way to do that. 

In addition, is it possible to use bootstrap typeahead on react-table?

Thank you!

**Comment by tannerlinsley** [comment]
This is definitely possible, but is also a user-land implementation thing. Happy to offer any guidance here in the form of question answer. :)

**Reply by jaekunchoi** [reply]
I am looking at same issue. I am trying to add multiple filters on same column. For example words column that has includes or excludes filter on that one column. @tannerlinsley are you able to guide me on this?

Thanks!
